std_msgs/Header header

sensor_msgs/NavSatFix gps

std_msgs/UInt64 casualty_id

std_msgs/String whisper
std_msgs/Float64 acconeer_respiration_rate
std_msgs/Header acconeer_respiration_rate_header
std_msgs/Float64 event_respiration_rate
std_msgs/Header event_respiration_rate_header
std_msgs/Float64 neural_heart_rate
std_msgs/Header neural_heart_rate_header
std_msgs/Float64 cv_heart_rate 
std_msgs/Header cv_heart_rate_header
std_msgs/Float64 neural_respiration_rate
std_msgs/Header neural_respiration_rate_header
